ui/raidboss/data/07-dt/alliance/cloud_of_darkness_chaotic.ts

```diff
@@ -0,0 +1,414 @@
+  // Vfx Path: m0684_charge_c2x
+  'pairStacks': '00F1',
+  // Stygian Shadow telegraph
+  // Vfx Path: m0684_charge_c3x
+  'proteanSpread': '00F2',
+  // tankbuster with circular cleave
+  // Vfx Path: tank_lockonae_4m_5s_01t
+  'tankCleave': '0156',
+  // flare marker
+  // Offsets: 76147,157739
+  // Vfx Path: all_at8s_0v
+  'flareMarker': '015A',
+  // player who will drop a bramble
+  // Vfx Path: m0302_seed_8s_x
+  'brambleSeed': '0227',
+  // countdown to hand spawning on player
+  // Offsets: 86301,97364,104403,111399,145339,153274,165355
+  // Vfx Path: l1rz_grasp_count5s_0x
+  'handCountdown': '0228',
+  // clockwise rotation
+  // Vfx Path: turnright_14m_14s_c0x
+  'rotateClockwise': '0234',
+  // counter-clockwise rotation
+  // Vfx Path: turnleft_14m_14s_c0x
+  'rotateCounterClockwise': '0235',
+  // bramble about to tether to a player
+  // Vfx Path: exclamation_8s_x
+  '0239': '0239',
+} as const;
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'TheCloudOfDarknessChaotic',
+  zoneId: ZoneId.TheCloudOfDarknessChaotic,
+  timelineFile: 'cloud_of_darkness_chaotic.txt',
+  initData: () => ({
+    deadlyEmbraceDodgeDirectionCollected: 'unknown',
+    deadlyEmbraceDodgeDirection: 'unknown',
+  }),
+  timelineTriggers: [],
+  triggers: [
+    {
+      id: 'Cloud Chaotic Inner/Outer Darkness Gain',
+      type: 'GainsEffect',
+      netRegex: { effectId: ['1051', '1052'], capture: true },
+      condition: Conditions.targetIsYou(),
+      run: (data, matches) => {
+        switch (matches.effectId) {
+          case '1051':
+            data.innerDarkness = true;
+            break;
+          case '1052':
+            data.outerDarkness = true;
+            break;
+        }
+      },
+    },
+    {
+      id: 'Cloud Chaotic Inner/Outer Darkness Lose',
+      type: 'LosesEffect',
+      netRegex: { effectId: ['1051', '1052'], capture: true },
+      condition: Conditions.targetIsYou(),
+      run: (data, matches) => {
+        switch (matches.effectId) {
+          case '1051':
+            delete data.innerDarkness;
+            break;
+          case '1052':
+            delete data.outerDarkness;
+            break;
+        }
+      },
+    },
+    {
+      id: 'Cloud Chaotic Doom',
+      type: 'GainsEffect',
+      netRegex: { effectId: 'D24', capture: true },
+      condition: (data, matches) => {
+        // this can get very noisy if we trigger on every player that receives Doom,
+        // so limit to only calling for players in your immediate party.
+        return data.CanCleanse() && data.party.inParty(matches.target);
+      },
+      alertText: (data, matches, output) => {
+        return output.text!({ player: data.party.member(matches.target) });
+      },
+      outputStrings: {
+        text: {
+          en: 'Cleanse ${player}',
+          de: 'Reinige ${player}',
+          fr: 'Guérissez ${player}',
+          cn: '康复 ${player}',
+          ko: '${player} 디버프 해제',
+        },
+      },
+    },
```

Comment:
In early prog with a group that's entirely fresh that's more likely to happen, but it tends to drop to one or two individuals (per party at least) once people get a bit more comfortable.  I could drop the names for a generic trigger, but I think I want to keep the limitation of only calling for your immediate party, since the alliance raid frames are frankly shit for seeing what's going on with the other parties quick enough to react.

Reply:
I agree that keeping it restricted to the current party is still good.